### PR TITLE
Closing ecosystem to MAC

### PR DESCRIPTION
**Pull request by SomeOne768**

### CODE REVIEW

File: Saint-Nec-Engine/CMakeLists.txt
Comment (SomeOne768) [inline Saint-Nec-Engine/CMakeLists.txt:1]: This ensure that the program can't compile on MAC OS

File: Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp
Comment (SomeOne768) [inline Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp:1]: This one ensure, program can't be run on MAC OS

File: Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp [hunk 1]
```diff
@@ -85,6 +85,11 @@ void draw_imgui(bool& show_demo_window, bool& show_another_window, ImVec4& clear
 
 int main(void)
 {
+    #ifdef __APPLE__
+        std::cerr << "Ecosysteme is closed for Apple users." << std::endl;
```
Comment (antonindevidal) [inline Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp]: Ecosystem*

File: Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp
Comment (antonindevidal) [reply Saint-Nec-Engine/Saint-Nec-Engine/src/Saint-Nec-Engine.cpp:1]: Maybe change text in english

**Review by antonindevidal — CHANGES_REQUESTED** [review]

**Pull request merged**